src/components/back-to-top/BackToTop.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+class BackToTop {
+  constructor() {
+    this._settings = {
+      selector: BackToTop.selector
+    }
+
```

Comment:
@yone920 Somewhere here, I think you should include a check to see if the selector exists on the page. If it doesn't, then return because none of the listeners should continue listening.